.github/workflows/docs.yml

```diff
@@ -0,0 +1,29 @@
+# A GitHub Actions workflow that generates the documentation for pull requests
+
+name: Documentation
+
+on:
+  push:
+
+jobs:
+  docs:
+    runs-on: ubuntu-20.04
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Install dependencies
+        run: npm ci
+
+      - name: Generate docs
+        run: npm run docs
+
+      - name: Commit docs
+        uses: EndBug/add-and-commit@v9
```

Comment:
Will this still commit if there are no changes?

Reply:
I asked kind the same question to @nmattia earlier and he said it will be processed but it will have no effect - i.e. no commit if nothing to be commited.

Reply:
Ok, great then. I'd rather run that locally so that we don't get an extra commit. Sometimes the extra commit from the CI messes up the CI.

Reply:
You can still run it locally but I personally, rather like not having to care about it. The all idea is to just care about documenting the code and that the rest happens magically automatically.

Reply:
Also the CI should be fine, it "just" commit/push in readme files, won't do anything else. Also there is such a job in II.

Being said, I'm not a devops so if anything need to be improved in the actions, more than happy to do so

Reply:
I have seen problems with the pipeline in nns-dapp when the format creates a commit. That's the reason.

Reply:
@lmuntaner if you notice issue with the pipeline after we merge this please tell and then we remove it.

thanks for the review